Trappist/src/Promact.Trappist.Utility/Constants/IStringConstants.cs

```diff
@@ -50,5 +50,12 @@ public interface IStringConstants
         /// </summary>
         string InvalidPasswordFormatError { get; }
         #endregion
+
+        #region "Category Constants"
+        /// <summary>
+        /// Property CatehoryNameError is used to check  CategoryName unquiness 
```

Comment:
CategoryNameError does not match with its property name. Plus statement is not properly formatted. space appeared twice after check word.